__tests__/app.test.js

```diff
@@ -15,28 +16,40 @@ beforeEach(() => seed(data));
 afterAll(() => db.end());
 
 describe("/api", () => {
-	describe("GET: 200 /topics", () => {
-		test("should respond with status code 200", () => {
-			return request(app).get("/api/topics").expect(200);
-		});
-		test("should respond with an array of topic objects, each of which should have the following properties: slug, description", () => {
-			return request(app)
-				.get("/api/topics")
-				.expect(200)
-				.then((response) => {
-					const { topics } = response.body;
-					expect(Array.isArray(topics)).toBe(true);
-					topics.forEach((topic) => {
-						expect(topic).toHaveProperty(
-							"slug",
-							expect.any(String)
-						);
-						expect(topic).toHaveProperty(
-							"description",
-							expect.any(String)
-						);
+	test("GET: 200 should respond with an object object describing all the available endpoints on this API", () => {
+		fs.readFile("./endpoints.json", "utf8")
+			.then((fileContents) => {
+				return JSON.parse(fileContents);
+			})
+			.then((expected) => {
+				return request(app)
+					.get("/api")
+					.expect(200)
+					.then((response) => {
+						const { endpoints } = response.body;
+						expect(endpoints).toMatchObject(expected);
```

Comment:
would like you to hardcode what the endpoint will look like instead of using fs.readfile because this test will auto pass for any subsequent updates

Reply:
Since the content of endpoint.json is dynamic, and the only consistent key on each endpoint is "description" I have added a `for in` to test for this in each endpoint object. 
I have also included a test to verify that the object I am iterating over in non-zero.